app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    copy_command, name, path = command.split()
```

Comment:
move_file != copy_command
Also is it really copy_command? 
If I run your code with command == "hello main.py goodbye" I will move my file main.py to file goodbye